app-vcs/git-credential-oauth/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=git-credential-oauth
+PKGSEC=vcs
+PKGDEP="glibc git"
+BUILDDEP="go"
+PKGDES="Git credential helper that securely authenticates to GitHub, GitLab, BitBucket, Gerrit and other forges using OAuth"
+ABSPLITDBG=0
```

Comment:
Why?

Reply:
This is a pure Go executable, so as documented this option should be turned off.

> For packages solely consisting of rust or go programs, you may want to turn OFF `ABSPLITDBG` [...]
> 
> -- <https://wiki.aosc.io/developer/packaging/autobuild3-manual/#the-defines-file>

Reply:
> This is a pure Go executable, so as documented this option should be turned off.
> 
> > For packages solely consisting of rust or go programs, you may want to turn OFF `ABSPLITDBG` [...]
> > -- https://wiki.aosc.io/developer/packaging/autobuild3-manual/#the-defines-file

This seems to be an out-of-date documentation. cc @chenx97 

Please try without this option.